### PR TITLE
Update setup.py, bump version

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
         'License :: OSI Approved :: Apache Software License',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```
Comment (kba) [inline setup.py]: I'm only testing with Python 3.5, Travis runs 3.4. Not sure how these classifiers are actually used but best be specific.

Comment (zuphilip) [inline setup.py]: Then I guess we should update Travis to run also on 3.5 and be more specific. Right?

**Comment by zuphilip** [comment]
Yes, the two PR by @stweil makes sense to wait for (and they should be almost ready to merge). I will do the tagging again in GitHub.

File: setup.py (hunk 1 above)
Comment (kba) [inline setup.py]: No, I would just be more specific with the classifiers, i.e.

```
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
 ```

We could also test with more versions on Travis. Reportlab supports 2.7 and 3.4 officially.

Comment (zuphilip) [inline setup.py]: Done.

**Comment by zuphilip** [comment]
Okay, all the work has been done now and we are ready for the release here.

**Comment by kba** [comment]
On [pypi](https://pypi.python.org/pypi?:action=display&name=hocr-tools&version=1.1.0)

**Comment by zuphilip** [comment]
Thank you very much! 🎉

**Pull request merged**